### DISCUSSION TITLE
Pausable/Resumable plots

### DISCUSSION THREAD

**Discussion by richardtheb** [Chia-Network/chia-blockchain#4972] [Ideas]
I write as someone who, like many users, uses their computers for more than one thing. I have a Big-Ass:tm: server that I use for video editing, but also for Chia plotting in off-hours. At the moment, I have to kill the chia processes and abandon the work when I want to start editing. which is a real waste. 

If instead, there was a way to pause plotting, I would be able to pause, get some work done, and then resume plotting afterward. 

I can't see a technical reason why it should not be possible: the work of plotting is, by definition, saved to disk, so it should be fesiable. 

Thoughts?

**Comment by ilyakooo0** [comment]
Please see this [wiki page](https://github.com/Chia-Network/chia-blockchain/wiki/FAQ#my-computerdrive-went-into-power-save-or-rebooted-while-plotting).

Yes, it is definitely possible to add support for resuming plots. 

The only reason it is not possible at the moment is that the main chia developers have not implemented it yet. Anyone is welcome to add support themselves and create a PR if they need support sooner. (:
